src/relay/App.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import Setting from "./components/Setting";
+
+function App(props: Data) {
```

Comment:
Typical react style would be to bind the keys in the function signature:
```suggestion
function App({ settings, rev }: Data) {
```